docs/reference/language-reference/built-in-types/int.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Int"
+sidebar_label: "Int"
+sidebar_position: 2
+keywords:
+  - types
+  - integer
+  - unsigned
+  - number
+  - long
+draft: false
+---
+
+Integers are, by default, 64 bit signed integers.
```

Comment:
I'd word this slightly differently, as it makes it sound like the int size is configurable. Maybe "default integers" or something would read slightly differently.

Reply:
I'm not quite following what you mean. Is the size not configurable by the suffix?